pyworklistserver/worklist.py

```diff
@@ -40,8 +40,8 @@ def _random_unicode_string(length):
     return ''.join(random.choices(' ' + string.ascii_uppercase + string.ascii_lowercase + string.digits + __NONASCII, k=length))
 
 def _clean_unicode_string(length):
-    """ Create a random string of specified length containing some non-ascii characters """
-    return ''.join(random.choices(' ' + string.ascii_uppercase + string.ascii_lowercase + string.digits + __NONASCII, k=length))
+    """ Create easily manageable string of specified length """
```

Comment:
When I think about it, is the function really creating a 'unicode string' as promised by the function name? Maybe it is better to call the function _create_random_ascii_string, and then the comment could explain that it "creates an ascii string with characters only in the range [0, 127].